python/pyiceberg/table/metadata.py

```diff
@@ -103,12 +109,12 @@ def construct_refs(cls, data: Dict[str, Any]):
     """The table’s base location. This is used by writers to determine where
     to store data files, manifest files, and table metadata files."""
 
-    table_uuid: Optional[UUID] = Field(alias="table-uuid", default_factory=uuid4)
+    table_uuid: uuid.UUID = Field(alias="table-uuid", default_factory=uuid.uuid4)
     """A UUID that identifies the table, generated when the table is created.
     Implementations must throw an exception if a table’s UUID does not match
     the expected UUID after refreshing metadata."""
 
-    last_updated_ms: int = Field(alias="last-updated-ms")
+    last_updated_ms: int = Field(alias="last-updated-ms", default_factory=lambda: int(time.time() * 1000))
```

Comment:
Is there no way to get time in milliseconds in Python?

Reply:
I keep forgetting about that helper class, thanks!